jobs/sportpursuit-python-developers-x2-roles.html

```diff
@@ -0,0 +1,70 @@
+---
+title: Python Developers (x2 roles)
+company: SportPursuit
+url: https://www.sportpursuit.com/
+location: London, UK
+
+contract: Permanent
+contact:
+    name: Steve Blake
+    email: join@sportpursuit.co.uk
+    phone: # Phone number (optional)
+    appliction form: https://sportpursuit.peoplehr.net/Pages/JobBoard/Opening.aspx?v=3b21579e-16a4-4828-941e-91ac3f4ddc5b
+created: !!timestamp 2018-04-25
+tags:
+  - python
+  - sql
+  - django
+  # include relevant tags
```

Comment:
Please use the tags feature to list major technologies that this role will need.